### PR TITLE
PNI styling updates

### LINKED ISSUE #3732: Update PNI product page styling
master issue: #3716

Comps: https://redpen.io/p/aj7f2e09f704b2e4ee

# Styling changes:

## **Remove**
- [ ] Remove fields listed [here](https://docs.google.com/spreadsheets/d/1tcFlDgrxxo4Mj5QP-pCFyB4QSCuPMi5vumLQu_r5-QI/edit#gid=0)
- [ ] Remove all accordions except for new accordion for Minimum Security Standards
- [ ] Remove pink info button and 'Read our research first' jump link in the "Tell us what you think" creep-o-meter box
- [ ] Remove the title "Our Research" and the pink info button and "How to use this guide" jump link
- [ ] Remove icons for old "Can it spy on me"/"It uses" section
- [ ] Remove icons for "How to contact company" section
- [ ] Remove smiley faces (not the creep-o-meter) and red highlights

_Comp:_
remove

## **Add**
- [ ] Add new fields listed [here](https://docs.google.com/spreadsheets/d/1tcFlDgrxxo4Mj5QP-pCFyB4QSCuPMi5vumLQu_r5-QI/edit#gid=0)
- [ ] Add Review Date
- [ ] Add Minimum Security Standards section with stars
_Comp:_
Screen Shot 2019-10-09 at 2 16 22 PM
- [ ] Bold field titles and corresponding answers
_Comp:_
Screen Shot 2019-10-09 at 2 16 56 PM

- [ ] Only show "Can it spy on me"/"It uses" items if the answer is Yes or Unknown _(might be done in https://github.com/mozilla/foundation.mozilla.org/issues/3733)_
- [ ] Hide "Parental controls?" if answer is N/A (might be done in #3733)
- [ ] Hide "Collects biometric data?" if answer is N/A (might be done in #3733)
- [ ] Add new styling to "What could happen if something goes wrong" section
    - [ ] Contain section in a div with a gradient border
    - [ ] Creep-o-meter face should break out of the border with about 16px of spacing on its left and right side
    - [ ] Center align the copy inside the div
     _Comp:_
Screen Shot 2019-10-09 at 2 17 43 PM

- [ ] "How to contact the company" should be body.normal instead of H3 to downplay the section
_Comp:_
Screen Shot 2019-10-09 at 2 18 23 PM

- [ ] Update copy to match comps

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #3732

Edit: https://foundation-mofostaging-pr-3788.herokuapp.com/cms/buyersguide/product/edit/64/
Page: https://foundation-mofostaging-pr-3788.herokuapp.com/en/privacynotincluded/products/vivofit-jr-2/

Notes: there are some font weight inconsistencies in the Sketch file... I'm not sure if those were made intentionally. I altered a few in this PR for the sake of consistency but let me know I should just follow what's in the Sketch file.

### CODE REVIEW

**Comment by Pomax** [comment]
Parental controls and Biomertric data hiding on "not applicable" was done in #3733

**Comment by Pomax** [comment]
From Slack: the star CSS needs a tweak so that value-yes = black, value-no/value-u = grey, and value-na = half (the scss I wrote maps yes to black, no to grey, unknown to half, and nothing for not applicable)

**Comment by kristinashu** [comment]
Sorry, when I put real content in I broke the link to the test page - it's here now https://foundation-mofostaging-pr-3788.herokuapp.com/en/privacynotincluded/products/vivofit-jr-2/

This is great! I don't think there are any critical blockers so I'll approve the PR. 

There are a few styling things, mostly around vertical spacing but again, there are not critical and can be done as a follow up.
- update `Minimum Security Standards` text to .body-large (it was something in between body small and large in the sketch but I think we can just do large)
- stars need a little more space between eachother
-  in general, more vertical space is needed between the diving lines and the text
-  in general, text should be vertically centered between the diving lines even when there is no help text
-  add diving line below `User friendly privacy info?` and 😮

Here's a dodgy comp trying to show some of these:

Screen-Shot-2019-10-16-at-6 27 56-PM

@beccaklam could you please take a look at this as well and add any issues you see? If you find something, please point out what are important issue and what is a nice to have.

**Comment by kristinashu** [comment]
Also, this section looks so good! Love the gradient.

image

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 1]
```diff
@@ -73,53 +79,44 @@ <h2 class="h1-heading col-md-10">{{product.name}}</h2>
...
-              {% endif %}
-            {% endfor %}
-          </ul>
-          {% endif %}
+          {% include "fragments/product_privacy.html"    with  label="User friendly privacy info?"  value=product.user_friendly_privacy_policy         links=product.privacy_policy_links         %}
         </div>
 
-        <div class="worst-case">
-          <h3 class="h3-heading"><span class="creep-face-emoji">😮</span>What could happen if something went wrong</h3>
+        <div class="worst-case-box px-4 px-md-5 pt-5 pb-2 pb-md-4 mb-5 text-center">
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/product_page.html]: now _that's_ a bootstrap class

File: source/sass/buyers-guide/views/product.scss [hunk 2]
```diff
@@ -89,13 +92,32 @@ $product-detail-padding-x: (
...
     font-family: "Nunito Sans";
   }
 
-  .creep-face-emoji {
-    top: 3px;
-    margin: 0 0.25em 0 0;
+  .worst-case-box {
+    position: relative;
+    border-width: 1px;
+    border-image: linear-gradient(to left, $pni-pink 0%, $pni-blue 100%) 1 / auto;
```
Comment (Pomax) [inline source/sass/buyers-guide/views/product.scss]: it looks like Chrome doesn't like this, but if I add the `-webkit-` prefix it looks like Chrome forgets what a border is, and just makes it the entire background O_O

image

**Comment by beccaklam** [comment]
@mmmavis The page looks great! +1 to @kristinashu's changes and aside from +1 @Pomax 's find above ☝️ with the missing gradient in chrome, I found two little things:
- [ ] We're missing an hr under _User friendly privacy info?_ section:
### PR:
Screen Shot 2019-10-17 at 11 05 19 AM
### Comp:
Screen Shot 2019-10-17 at 10 59 20 AM
I don't know if it's completely necessary but I feel like it "closes off" the section better. @kristinashu Thoughts?
- [ ] Also for the Minimum Security Standards section, there's still some spacing that's showing up under the section title even when there's no elaboration text. Would it be possible to get ride of that?
### PR:
Screen Shot 2019-10-17 at 11 02 50 AM

File: source/sass/buyers-guide/views/product.scss (hunk 2 above)
Comment (Pomax) [reply source/sass/buyers-guide/views/product.scss]: The rule specified in https://css-tricks.com/gradient-borders-in-css/ does seem to work in both Chrome and Firefox

**Comment by mmmavis** [comment]
Updated the PR with the following. Gonna leave other styling tweaks mentioned above as follow-up.

- fixed gradient border bug on Chrome
- add divider line around the `User friendly privacy info?` section
- update `Minimum Security Standards` text to .body-large
- increase spacing in between the stars

File: source/sass/buyers-guide/views/product.scss (hunk 2 above)
Comment (mmmavis) [reply source/sass/buyers-guide/views/product.scss]: Thanks for the link @Pomax . It turned out just adding `border-style: solid` fixed the Chrome issue. Phew! One fewer divs to maintain 😂

**Comment by kristinashu** [comment]
Nice! Those are looking good!

**Comment by Pomax** [comment]
Sweet!

**Comment by kristinashu** [comment]
I've opened a ticket for the remaining styling tweaks https://github.com/mozilla/foundation.mozilla.org/issues/3793

Hopefully this is it but I would like to review it again and more thoroughly once it's on staging.

**Comment by beccaklam** [comment]
> Nicely spotted @Pomax!
> 
> And thank you @beccaklam! I did mention those things but definitely not as clearly or nicely as you have. @mmmavis let me know if you need clarification on any of these.

Lol oops 😅 Definitely skim-read that too quickly.

**Pull request merged**